packages/workbox-webpack-plugin/build.js

```diff
@@ -17,6 +17,6 @@ const fse = require('fs-extra');
 const path = require('path');
 
 module.exports = () => {
-  return fse.copy(path.join(__dirname, 'index.js'),
-    path.join(__dirname, 'build', 'index.js'));
+  return fse.copy(path.join(__dirname, 'src'),
```

Comment:
Is this needed?

I think this may have been a thing that was needed for older versions but the build model has changed in V3. At the moment lerna will just publish the directoy as is and won't use build at all.

Reply:
I've got a build process working in v3 for Node modules is this PR: https://github.com/GoogleChrome/workbox/pull/811

(As of right now, it determines that something is a Node module based on the presence of `engines.node` in `package.json`, which is a bit of magic.)

In summary, assuming #811 is merged first and this PR is rebased against the `v3` branch, you should be able to get rid of this `build.js` file entirely.

Reply:
The goal is to get #811 reviewed and merged into `v3`, yes. So that would ultimately be what this PR would build on top of. I'll modify the code in that PR to address some of the things you've asked for.

Reply:
With #811 now merged into v3, now might be a good time to try rebuilding this PR on top of what is there. It's unclear to me how much churn we might see in the workbox-build interface changes but @jeffposnick seemed to be supportive of either rebasing against v3 or waiting a little longer. I think we're fine with whatever your preference is, @goldhand.